cloudflare/resource_cloudflare_fallback_domain.go

```diff
@@ -3,11 +3,20 @@ package cloudflare
 import (
 	"context"
 	"fmt"
+	"log"
 
 	cloudflare "github.com/cloudflare/cloudflare-go"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// use this since we cannot use const for lists.
+func default_domains() []string {
```

Comment:
not sure why the linter didn't pick this up but we should use camel casing in Go where the method isn't exported.

```suggestion
func fallbackDefaultDomains() []string {
```

Reply:
alternatively, a `var` will also work here.